test/unit/session.spec.js

```diff
@@ -71,7 +71,6 @@ describe('util/session', () => {
     it('sends the correct request', () => {
       testData.extendedUsers.createPast(1);
       setData({ session: testData.sessions.createNew({ token: 'foo' }) });
-      store.commit('setSendInitialRequests', false);
```

Comment:
I realized while writing the new tests that this line doesn't seem to be needed. Maybe it used to be, but I changed something in #420 such that it's no longer needed? I think I used to use it in any test that ran `logOut()` after calling `mockHttp()` (rather than `load()`). But there's no navigation in that case, so I don't think it's needed. (If there were navigation, the router would send the initial requests, and `mockHttp()` would throw an error about a request without a response.) Right now the tests use `load()` when there's navigation.